src/Distributions.jl

```diff
@@ -230,6 +234,7 @@ export
     rate,               # get the rate parameter
     sampler,            # create a Sampler object for efficient samples
     scale,              # get the scale parameter
+    scale!,             # provide storage for the scale parameter (used in multivariate distribution mvlognormal)
```

Comment:
Note that `scale` and `scale!` are both defined in base, but with a different meaning (that of scaling a vector or matrix, as opposed to extracting a vector).

It's probably okay here (especially since `scale` has been defined, but in general, having the same function refer to different concepts is discouraged in Julia.  

@StefanKarpinski, I often want to cite one of your talks or an issue when saying this, but I can't find a good one--any suggestions?  Assuming, of course, that I'm not misrepresenting the idea.

(@KrisDM, this probably has no direct impact on this PR, just things to keep in mind.)